JokeRandomizer/JokeRandomizer/CustomButtons/CustomButton.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  CustomButton.swift
+//  JokeRandomizer
+//
+//  Created by john.adrian.de.claro on 11/28/22.
+//
+
+import UIKit
+
+class CustomButton: UIButton {
+    
```

Comment:
you can put an enum here for constants and use this in the implementation instead of the static text

private enum Constants {
		enum SystemName {
			static let arrowClockwiseCircleFill = "arrow.clockwise.circle.fill"
			static let plus = "plus"
			static let listBulletCircle = "list.bullet.circle"
		}
	}

Reply:
Create a AppConstant File